### ISSUE TITLE
Create a demonstration video

### ISSUE THREAD

**Issue by matt-dray** [co-analysis/a11ytables#95]
There's a gptables video on [the resources page](https://analysisfunction.civilservice.gov.uk/policy-store/further-resources-for-releasing-statistics-in-spreadsheets/) for releasing statistics in spreadsheets, which shows how to use it. Maybe there should be one for {a11ytables}? There are videos from talks, but maybe something more technical, like a screen recording of RStudio.

**Comment by matt-dray** [comment]
I think there's sufficient documentation out there; could do this if requested.
